source/pic2card/.pylintrc

```diff
@@ -143,7 +143,9 @@ disable=print-statement,
         xreadlines-attribute,
         deprecated-sys-function,
         exception-escape,
-        comprehension-escape
+        comprehension-escape,
+        consider-using-with,
+        relative-beyond-top-level
```

Comment:
we have to reduce the number of items in this list, let's plan for it after the current activities.